full-stack-tests/core/webpack.config.js

```diff
@@ -45,6 +45,12 @@ function createConfig(shouldInstrument) {
         stream: require.resolve("stream-browserify"),
         zlib: require.resolve("browserify-zlib"),
       },
+      alias: {
+        "@azure/storage-blob$": "@azure/storage-blob/dist-esm/storage-blob/src/index.browser.js",
+        "@azure/core-http$": "@azure/core-http/dist-esm/src/coreHttp.js",
+        "@azure/logger$": "@azure/logger/dist-esm/src/index.js",
+        "supports-color$": "supports-color/browser.js"
```

Comment:
@pmconne @wgoehrig I had to add another 2 lines to make it build. I don't think this warrants a whole new round of code review, but please resolve this comment if you agree with this change. I'll leave the PR on auto-complete.